sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -880,25 +880,27 @@ SIREPO.app.directive('logoutMenu', function(authState, authService, requestSende
     return {
         restrict: 'A',
         scope: {},
-        template: [
-            '<li data-ng-if="::authState.isGuestUser"><a data-ng-href="{{ ::authService.loginUrl }}"><span class="glyphicon glyphicon-alert sr-small-icon"></span> Save Your Work!</a></li>',
-            '<li data-ng-if="::! authState.isGuestUser" class="sr-logged-in-menu dropdown">',
-              '<a href class="dropdown-toggle" data-toggle="dropdown">',
-                '<img data-ng-if="::authState.avatarUrl" data-ng-src="{{:: authState.avatarUrl }}">',
-                '<span data-ng-if="::! authState.avatarUrl" class="glyphicon glyphicon-user"></span>',
-                ' <span class="caret"></span>',
-              '</a>',
-              '<ul class="dropdown-menu">',
-                '<li class="dropdown-header"><strong>{{ ::authState.displayName }}</strong></li>',
-                '<li class="dropdown-header">{{ authState.paymentPlanName() }}</li>',
-                '<li class="dropdown-header" data-ng-if="::authState.userName">{{ ::authState.userName }}</li>',
-                '<li data-ng-if="showAdmJobs()"><a data-ng-href="{{ getUrl(\'admJobs\') }}">Admin</a></li>',
-                '<li><a data-ng-href="{{ getUrl(\'ownJobs\') }}">Jobs</a></li>',
-                '<li><a data-ng-href="{{ ::authService.logoutUrl }}">Sign out</a></li>',
-              '</ul>',
-            '</li>',
-        ].join(''),
-        controller: function($scope) {
+        template: `
```

Comment:
Thanks for converting to new style:)

Reply:
3f4d10d86518e0e428966ed7697717e1d82d6e96